addon/devtools/loader.js

```diff
@@ -10,24 +10,39 @@ if (chrome.devtools?.recorder) {
     stringify(recording) {
       const autoconsentSteps = recording.steps
         .filter((step) => ["click"].includes(step.type))
-        .map(({ selectors }) => {
-          return {
-            waitForThenClick: selectors,
-          };
-        });
+        .map(({ selectors }) => this.convertStep(selectors));
       return Promise.resolve(JSON.stringify(autoconsentSteps, undefined, 4));
     }
     stringifyStep(step) {
       return Promise.resolve(
         JSON.stringify(
-          {
-            waitForThenClick: step.selectors,
-          },
+          this.convertStep(step.selectors),
           undefined,
           4
         )
       );
     }
+    convertStep(selectors) {
+      return {
+        any: selectors.filter((s) => {
+          return s.every(sel => this.isSupportedSelector(sel))
+        }).map(s => {
+          if (s.length === 1) {
+            return {
+              waitForThenClick: s[0]
+            }
+          }
+          return {
+            waitForThenClick: s
```

Comment:
`waitForThenClick` does not support array as an input. Does `s` ever have more than one element?

Reply:
Yes. The recorder can generate chained selectors, e.g. `['step1', 'step2']`, which should get unwrapped to `document.querySelector('step1').querySelector('step2')`.